Shared/UserAgent.swift

```diff
@@ -85,4 +85,26 @@ public class UserAgent {
 
         return firefoxUA
     }
+
+    public static func desktopUserAgent(defaults: NSUserDefaults) -> String {
+        let userAgent = NSMutableString(string: defaultUserAgent(defaults))
+
+        // Spoof platform section
+        let platformRegex = try! NSRegularExpression(pattern: "\\([^\\)]+\\)", options: [])
+        guard let platformMatch = platformRegex.firstMatchInString(userAgent as String, options:[], range: NSMakeRange(0, userAgent.length)) else {
+            print("Error: Unable to determine platform in UA.")
+            return String(userAgent)
+        }
+        userAgent.replaceCharactersInRange(platformMatch.range, withString: "(Macintosh; Intel Mac OS X 10_11_1)")
+
+        // Strip mobile section
+        let mobileRegex = try! NSRegularExpression(pattern: " FxiOS/[^ ]+ Mobile/[^ ]+", options: [])
```

Comment:
It feels a bit weird to strip the FxiOS section here because it is added in `defaultUserAgent` which is called before. Would it be worth to extract a private method with the common parts of `defaultUserAgent` and `desktopUserAgent`? If so, I think we'd also have to adapt the caching.